libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -227,28 +233,39 @@ void AP_Baro::calibrate(bool save)
    update the barometer calibration
    this updates the baro ground calibration to the current values. It
    can be used before arming to keep the baro well calibrated
+
+   The GPS is used to estimate and corect foour deviation from an ISA day.
```

Comment:
Ahahah I usually don't look at commit messages with the same detail, so you're safe. By the way, _co**r**rect_ is also wrong :smile: